### ISSUE TITLE
Readme docs ... additional user scenario

### ISSUE THREAD

**Issue by EmmaJP** [bbc/bbc-a11y#255]
Some developers will use bbc-a11y from command line, if the project does not have a compatible CI. Suggest the following sets of docs  under "Using bbc-a11y" for the readme (appropriately worded so as not to scare less technical users, or put off more experienced users):

- Basics (/guides/using/checking-a-website.md)
- Experienced (basics + other configeration that will work in commandline mode in /guides/using/using-bbc-a11y-in-your-project.md)
- Continuous Integration (basics + experienced + remaining stuff in /guides/using/using-bbc-a11y-in-your-project.md)
- CI with Docker (/docker/README.md)

Also need to change wording in/around link to coverage so the link text is clearer out of context.

**Comment by EmmaJP** [comment]
Do #229 at same time
